manubot/process/header-includes-template.html

```diff
@@ -0,0 +1,77 @@
+{% endif %}
+{% if iso_date is defined %}
+<meta name="dc.date" content="{{ iso_date }}">
+<meta name="citation_publication_date" content="{{ iso_date }}">
+{% endif %}
+{% if lang is defined %}
+<meta name="dc.language" content="{{ lang }}">
+<meta name="citation_language" content="{{ lang }}">
+{% endif %}
+<meta name="dc.relation.ispartof" content="Manubot Preprint">
+<meta name="dc.publisher" content="Manubot">
+<meta name="citation_journal_title" content="Manubot Preprint">
+<meta name="citation_technical_report_institution" content="Manubot">
+<!-- author information --> 
+{% for author in authors|default([]) if author.name is defined %}
+<meta name="citation_author" content="{{ author.name|escape }}">
+{% if author.affiliations is defined %}
+{% for affiliation in author.affiliations %}
+<meta name="citation_author_institution" content="{{ affiliation|escape }}">
+{% endfor %}
+{% endif %}
+{% if author.orcid is defined %}
+<meta name="citation_author_orcid" content="{{ author.orcid }}">
+{% endif %}
+{% if author.twitter is defined %}
+<meta name="twitter:creator" content="@{{ author.twitter }}">
+{% endif %}
+{% endfor %}
+{% if html_url is defined %}
+<link rel="canonical" href="{{ html_url }}">
+<meta property="og:url" content="{{ html_url }}">
+<meta property="twitter:url" content="{{ html_url }}">
+<meta name="citation_fulltext_html_url" content="{{ html_url }}">
+{% endif %}
+{% if pdf_url is defined %}
+<meta name="citation_pdf_url" content="{{ pdf_url }}" />
+<link rel="alternate" type="application/pdf" href="{{ pdf_url }}">
+{% endif %}
+{% if html_url_versioned is defined %}
+<link rel="alternate" type="text/html" href="{{ html_url_versioned }}">
+<meta name="manubot_versioned_html_url" content="{{ html_url_versioned }}">
+{% endif %}
+{% if pdf_url_versioned is defined %}
+<meta name="manubot_versioned_pdf_url" content="{{ pdf_url_versioned }}">
+{% endif %}
+{% if description is defined %}
+<meta name="description" content="{{ description|escape }}">
+<meta name="dc.description" content="{{ description|escape }}"/>
+<meta property="og:description" content="{{ description|escape }}">
+<meta property="twitter:description" content="{{ description|escape }}">
+{% endif %}
+{% if abstract is defined %}
+<meta name="dc.abstract" content="{{ abstract|escape }}">
+<meta name="citation_abstract" content="{{ abstract|escape }}">
+{% endif %}
+<meta property="og:type" content="article">
+<meta property="twitter:card" content="summary_large_image">
+{% if thumbnail_url is defined %}
+<meta property="og:image" content="{{ thumbnail_url }}">
+<meta property="twitter:image" content="{{ thumbnail_url }}">
+{% endif %}
+<!-- favicons from manubot.org -->
+<link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png">
+<link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457">
+<link rel="apple-touch-icon" sizes="180x180" href="https://manubot.org/apple-touch-icon.png">
```

Comment:
```
<link rel="manifest" href="/site.webmanifest">
<meta name="msapplication-TileColor" content="#ad1457">
```

Reply:
Or consider keeping just application-agnostic settings like icon, mask-icon and theme-color.

Reply:
> Or consider keeping just application-agnostic settings like icon, mask-icon and theme-color.

￼done in https://github.com/manubot/manubot/pull/138/commits/521b551e9554ba3edb1bd45703fa33a4382babd2